manim/mobject/mobject.py

```diff
@@ -543,7 +593,29 @@ def generate_target(self, use_deepcopy=False):
 
     # Updating
 
-    def update(self, dt=0, recursive=True):
+    def update(self, dt=0, recursive=True) -> "Mobject":
+        """Apply all updaters.
+
+        Does nothing if updating is suspended.
+
+        Parameters
+        ----------
+        dt : :class:`float`
```

Comment:
whichever function you have typings you can remove the parameter type in docstring it will be automatically added
see https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html